src/style/Color.js

```diff
@@ -100,6 +100,7 @@ var Color = Base.extend(new function() {
                     data[1] / 255,
                     data[2] / 255
                 ];
+                if (data[3] < 255) cached.push(data[3] / 255);
```

Comment:
I don’t really see one-liner `if`s in the rest of the code, I see ones without curly’s though. Maybe good to match the existing style?

Reply:
there are a few ... eg line 183